tromega/lib/views/account/Example/user.dart

```diff
@@ -0,0 +1,18 @@
+class User {
+  final String imagePath;
+  final String name;
+  final String age;
+  final String height;
+  final String weight;
+  final String gender;
+  final String goal;
+
+  const User(
+      {required this.imagePath,
+      required this.name,
+      required this.age,
+      required this.height,
+      required this.weight,
+      required this.gender,
+      required this.goal});
+}
```

Comment:
Bitte hier einen Konstruktor anlegen und nicht eine const von User:
User(this.imgPath, this.name ...usw)